### PR TITLE
Add database configuration document to new wiki

### LINKED ISSUE #5893: [Docs] Update docs Post 22 release
[placeholder issue for documentation that should be fixed asap since the 22.0.0 release changes]

since https://github.com/aces/Loris/pull/5785 : `config.xml` now has a placeholder for a db `adminUser` credential (renamed from `quatuser`). Needs documentation, including: 
* existing projects will need to update (This PR was supposed to be noted as a caveat for Existing Projects in the 22 release notes, to enable them to run scripts.  
* which privileges it should have
* should never be `root`, because it's web-exposed

### LINKED ISSUE #5919: Create Wiki document for MySQL user policy
**Describe the solution you'd like**
Creation of a very concise Wiki page that explains for MySQL user policy. 

Essentially we have two users, an everyday user with few permissions and an "admin" (formerly known as "quatUser") user that has GRANT and DROP table permissions. The latter should only be used by the install script and (temporarily) the instrument manager. 
 
**Additional context**
Discussion arose in #5253

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes
Add information about MySQL/MariaDB configuration to the new wiki/ReadTheDocs.

#### Link(s) to related issue(s)

* Resolves #5919 
* Resolves #5893

### CODE REVIEW

File: docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/Database Configuration.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+credential must be created before continuing. (This is not the same as a unix 
+        `root` credential.) 
+
+After running the install script, you will be prompted to visit the 
+`installdb.php` page at the IP address that LORIS is hosted at.
+
+Completing the form on this page will create a new database and user account
+that will be used to execute transactions coming from LORIS's PHP code.
+
+### Administrative Account
```
Comment (johnsaigle) [inline docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/Database Configuration.md]:
```suggestion
## Administrative Account
```

File: docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/Database Configuration.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Database Configuration
+
+## Creating the Database
+MySQL or MariaDB must be installed and a `root` or admin-level user 
+credential must be created before continuing. (This is not the same as a unix 
+        `root` credential.) 
+
+After running the install script, you will be prompted to visit the 
+`installdb.php` page at the IP address that LORIS is hosted at.
```
Comment (driusan) [inline docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/Database Configuration.md]: It should be at the canonical domain that it's to be hosted at, the install script uses the place that it's
accessed from in order to derive `$host`

Comment (johnsaigle) [inline docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/Database Configuration.md]:
```suggestion
`installdb.php` page at the canonical domain of your LORIS instance.
```

**Comment by johnsaigle** [comment]
@driusan @ridz1208 Could you merge this if satisfied?

**Pull request merged**